admin/data/sql/migrations/026.do.create-group.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE [mtc_admin].[group]
+(
+  [id] [int] IDENTITY(1,1) NOT NULL,
+  [name] [nvarchar](max) NOT NULL,
```

Comment:
@jon-shipley thanks for the UNIQUE INDEX suggestion. There is a validation in place to avoid group name duplicates, however strictly we can have duplicates because we do a soft-delete, so the validation checks on active group names.
Yes, a group is always linked to a particular school.

Reply:
Ok, thanks @pris54